pro/src/screens/OfferType/__specs__/OfferType.spec.tsx

```diff
@@ -162,7 +162,7 @@ describe('screens:OfferIndividual::OfferType', () => {
 
     expect(mockHistoryPush).toHaveBeenCalledWith({
```

Comment:
@Aliochka tu ne le vois pas forcément vu qu'il n'y a pas encore de commentaire de SonarCloud sur les PRs mais SonarCloud détecte de la duplication ici.

Effectivement ça peut être un use case pertinent de `it.each()` en bouclant sur les 4 types de l'enum